fix/test-user-validation (#2360)
mercadopago/devsite-docs: guides/snippets/test-integration/create-test-users.pt.md

> NOTE
>
> Nota
>

> > Para editar a **identificação da conta** ou **adicionar mais dinheiro fictício** para testar suas aplicações, clique nos **3 pontos verticais** ao final da linha da tabela e selecione a opção **Editar dados**.<br> <br> Você pode gerar **até 15 contas** de usuários de teste ao mesmo tempo e, por enquanto, ainda não é possível deletá-las.

Reviewer: ```suggestion
> Para editar a **identificação da conta** ou **adicionar mais dinheiro fictício** para testar suas aplicações, clique nos **3 pontos verticais** ao final da linha da tabela e selecione a opção **Editar dados**. É possível criar **até 15 contas** de usuários de teste simultaneamente, no entanto, ainda não é possível deletá-las.
```

Reviewer: @pcanosa - como boa prática de TW, devemos sempre evitar quebras de linhas em disclaimers. Disclaimers geralmente contêm informações críticas que precisam ser transmitidas de maneira rápida e clara. Evitar quebras de linha mantém o texto compacto e direto ao ponto.

Te passo mais contexto sobre isso depois :)

## Validar login com usuários teste

Ao realizar o login com usuários de teste e acessar algumas seções do Painel do Desenvolvedor, a autenticação da conta mediante um código enviado por e-mail pode ser solicitada.

Como se tratam de usuários fictícios, você não terá acesso ao e-mail que receberá o código. Em vez disso, você deve realizar essa autenticação inserindo os **últimos 6 dígitos que compõem o User ID da conta de teste** ou os **últimos 6 dígitos que compõem o Access Token produtivo**.

> WARNING
>
> Importante
>

> > Para acessar ao User ID ou o Access Token de uma conta de teste, você deve ter criado previamente uma aplicação. Para saber como fazer isso, consulte a documentação sobre o [Painel do Desenvolvedor](/developers/pt/docs/your-integrations/dashboard). <br><br> Se tiver dúvidas sobre como obter o User ID ou o Access Token, consulte [Detalhes da aplicação](/developers/pt/docs/your-integrations/application-details) ou [Credenciais](/developers/pt/docs/your-integrations/credentials).

Reviewer: ```suggestion
> Para acessar o User ID ou o Access Token de uma conta de teste, é necessário criar previamente uma aplicação. Para isso, consulte a documentação [Painel do Desenvolvedor](/developers/pt/docs/your-integrations/dashboard). Caso tenha dúvidas sobre como obter o User ID ou o Access Token, consulte [Detalhes da aplicação](/developers/pt/docs/your-integrations/application-details) ou [Credenciais](/developers/pt/docs/your-integrations/credentials).
```

Reviewer: Mesma coisa aqui, @pcanosa - não quebramos disclaimers  :)

> NOTE
>
> Nota
>
> Para editar a **identificação da conta** ou **adicionar mais dinheiro fictício** para testar suas aplicações, clique nos **3 pontos verticais** ao final da linha da tabela e selecione a opção **Editar dados**.<br> <br> Você pode gerar **até 15 contas** de usuários de teste ao mesmo tempo e, por enquanto, ainda não é possível deletá-las.

> 

Reviewer 2: ```suggestion
```
